weave/trace_server/clickhouse_trace_server_batched.py

```diff
@@ -319,6 +320,9 @@ def call_end(self, req: tsi.CallEndReq) -> tsi.CallEndRes:
         # the call does not already exist
         self._insert_call(ch_call)
 
+        kafka_producer = KafkaProducer.from_env()
```

Comment:
idk what the overhead is of creating a new producer & connection is for each call, but this might be better suited as a property of the server class itself.

Reply:
Fixed.